### ISSUE TITLE
Support mouse dragging

### ISSUE THREAD

**Issue by Drarig29** [Drarig29/brackets-viewer.js#39]
Could be cool to enable this (Toornament does it).

A track: https://github.com/kmturley/touch-scroll

---

Related to #7

**Comment by qlaffont** [comment]
In fact, you can easely fix it with simple CSS :
```
.bracket-viewer {
  overflow-x: scroll;
  width: 100%;
}
```

**Comment by Drarig29** [comment]
No, I mean really dragging with your mouse. Like if you move the mouse while holding the click, it will drag the view.

**Comment by Drarig29** [comment]
Just try [touch-scroll](https://github.com/kmturley/touch-scroll) and you'll understand 😉
